src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/cars/add")
```

Comment:
according to the checkstyle
```suggestion
@WebServlet(urlPatterns = "/cars/drivers/add")
```
<img width="1011" alt="Снимок экрана 2021-12-10 в 13 26 38" src="https://user-images.githubusercontent.com/79812194/145567159-6e4c35b8-d905-4595-9171-8511ecc36993.png">